doc/reference/overview.rst

```diff
@@ -239,6 +239,11 @@ current :ref:`stability level <api_lifecycle>`.
      - 1.11
      - 2.1
 
+   * - :ref:`flash_buffered_write`
+     - Experimental
+     - 2.2
```

Comment:
Shouldn't these both be 2.3?

Reply:
Probably, I had no idea what I was doing. Why is two versions required?

Reply:
Take a look here:

Reply:
And now it all makes sense, thank you :) I applied your proposal.